test/mdlinks.spec.js

```diff
@@ -0,0 +1,63 @@
+const {
+  absolutePath, walkDir, getLinks,
+} = require('../src/mdlink.js');
+
+describe('Funtion AbsolutePath', () => {
+  it('is a function', () => {
+    expect(typeof absolutePath).toBe('function');
+  });
+  it('Deberia retornar un string con la ruta absoluta', () => {
+    const output = 'C:\\Users\\eliza\\Documents\\MdLinks\\CDMX009-MdLinks\\src\\mdlink.js'; 
```

Comment:
si estuvieses trabajando con 5 programadoras más en el equipo ¿cual crees que sea el resultados si usas esta ruta para testear? les causara conflicto
```suggestion
    const output = 'C:\\Users\\eliza\\Documents\\MdLinks\\CDMX009-MdLinks\\src\\mdlink.js'; 
```